Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM alpine:3.6
+FROM alpine:3.14
```

Comment:
3.15 is available now. Since the base container is being updated, it might be good to use that version instead.